admin/bin/qserv-admin.py

```diff
@@ -154,7 +186,7 @@ def receiveCommands(self):
         cmd = ''
         prompt = self._prompt
         while True:
-            line = raw_input(prompt).decode("utf-8").strip()
+            line = raw_input(prompt).strip()
```

Comment:
As far as I can tell, the initial decode is bogus regardless of what goes on in the C++ layer - it can cause lookup failures for command names in the python layer. My suggestion aims to take care of arbitrary user encodings by first decoding to a unicode instance, then encoding back to a UTF-8 `str`, which is ASCII compatible. 

If you don't want to deal with any of this now, maybe file an issue to fix this so we don't forget?